config/_default/menus/menus.en.yaml

```diff
@@ -0,0 +1,1295 @@
+    url: "tracing/visualization/trace/"
+    parent: "tracing_vis"
+    weight: 404
+  - name: "Service Map"
+    url: "tracing/visualization/services_map/"
+    parent: "tracing_vis"
+    weight: 405
+
+  ## TRACING - Trace Search
+
+  - name: "Trace Search & Analytics"
+    url: "tracing/trace_search_and_analytics/"
+    parent: "tracing"
+    identifier: "tracing_search_analytics"
+    weight: 5
+  - name: "Agent Configuration"
+    url: "tracing/trace_search_and_analytics/agent_trace_search/"
+    parent: "tracing_search_analytics"
+    weight: 501
+  - name: "Trace Search"
+    url: "tracing/trace_search_and_analytics/search/"
+    parent: "tracing_search_analytics"
+    weight: 502
+  - name: "Trace Analytics"
+    url: "tracing/trace_search_and_analytics/analytics/"
+    parent: "tracing_search_analytics"
+    weight: 503
+  - name: "Monitors"
+    url: "monitors/monitor_types/trace_analytics"
+    parent: "tracing_search_analytics"
+    weight: 504
+
+  - name: "Trace API"
+    url: "api/#tracing"
+    identifier: "tracing api"
+    parent: "tracing"
+    weight: 6
+
+  - name: "Community Libraries"
+    url: "developers/libraries/#apm-tracing-client-libraries/"
+    identifier: "community_library"
+    parent: "tracing"
+    weight: 7
+
+  - name: "Guides"
+    url: "tracing/guide/"
+    identifier: "tracing_guides"
+    parent: "tracing"
+    weight: 8
+
+  - name: Troubleshooting
+    url: "tracing/troubleshooting/"
+    identifier: "tracing_troubleshooting"
+    parent: "tracing"
+    weight: 9
+
+##########
+## LOGS ##
+##########
+
+  - name: "Log Management"
+    url: "logs/"
+    pre: "nav_logs"
+    identifier: "log_management"
+    weight: 90000
+
+  ## LOGS -  Log Collection
+
+  - name: "Log Collection & Integrations"
+    url: "logs/log_collection/"
+    parent: "log_management"
+    identifier: "log_collection"
+    weight: 1
+  - name: "Browser"
+    identifier: "log_browser"
+    url: "logs/log_collection/javascript/"
+    parent: "log_collection"
+    weight: 101
+  - name: "Csharp"
+    url: "logs/log_collection/csharp/"
+    parent: "log_collection"
+    weight: 102
+  - name: "Go"
+    identifier: "log_go"
+    url: "logs/log_collection/go/"
+    parent: "log_collection"
+    weight: 103
+  - name: "Java"
+    url: "logs/log_collection/java/"
+    parent: "log_collection"
+    identifier: "log_collection_java"
+    weight: 104
+  - name: "Nodejs"
+    url: "logs/log_collection/nodejs/"
+    parent: "log_collection"
+    weight: 105
+  - name: "PHP"
+    identifier: "log_php"
+    url: "logs/log_collection/php/"
+    parent: "log_collection"
+    weight: 106
+  - name: "Python"
+    identifier: "log_python"
+    url: "logs/log_collection/python/"
+    parent: "log_collection"
+    weight: 107
+  - name: "Ruby"
+    url: "logs/log_collection/ruby/"
+    parent: "log_collection"
+    identifier: "log_collection_ruby"
+    weight: 108
+  - name: "Other Integrations"
+    url: "integrations/#cat-log-collection"
```

Comment:
I'm not sure if websites has access to test, but this link doesn't work on the Microsoft Edge browser.

Using the category buttons on the integrations page does work on Edge.

Reply:
fixed

Reply:
Thanks Zach. I'll run a full test on Edge later today.

Reply:
Edge looks good.